run_config_input.py

```diff
@@ -258,8 +253,6 @@ def run_normal_config_file(config_file="./examples/configuration.json"):
         model.__str__()
         + "-r-"
         + str(config["radius"])
-        + "-mnnn-"
-        + str(config["max_num_node_neighbours"])
```

Comment:
This probably is what causes the failure in testing. The model name in test_trained_model.py also needs updated/consistent with this one.

Reply:
Thanks! Fixed